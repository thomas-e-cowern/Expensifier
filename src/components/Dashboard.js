import React from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';
import ExpensesSummary from './ExpenseSummary';

const expenseDashboardPage = () => (
    <div>
        <ExpensesSummary />
        <ExpenseListFilters />
        <ExpenseList />
    </div>
);

export default expenseDashboardPage;