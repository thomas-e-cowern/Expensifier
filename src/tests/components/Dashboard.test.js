import React from 'react';
import {
    shallow
} from 'enzyme';
import Dashboard from '../../components/Dashboard';

test('Should render Dashboard corectly', () => {
    
    const wrapper = shallow(<Dashboard />);
    expect(wrapper).toMatchSnapshot();
});