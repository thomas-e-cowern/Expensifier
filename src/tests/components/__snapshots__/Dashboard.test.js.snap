// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Dashboard corectly 1`] = `
<div>
  <Connect(ExpenseSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
